accessapproval/CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 1.0.0
+
+Stabilize GA surfaces.
```

Comment:
Nit: I don't have a better way to phrase it, but it seems funny to have surfaces be plural when this is a single module. :man_shrugging:

Reply:
Updated to make singular.